bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -1641,6 +1641,19 @@ use_crop=".true.">lnd/clm2/surfdata_map/ctsm5.1.dev052/landuse.timeseries_mpasa1
 <soilm_tintalgo>linear</soilm_tintalgo>
 <soilm_offset  >0</soilm_offset>
 
+<!-- Irrigation streams namelist defaults -->
+<!-- This is if you want to prescribe the irrigation from input datasets rather than model it -->
+<use_irrigation_streams>.false.</use_irrigation_streams>
+<stream_year_first_irrig  >1850</stream_year_first_irrig>
+<stream_year_last_irrig   >1850</stream_year_last_irrig>
+<model_year_align_irrig   >1850</model_year_align_irrig>
+
+<stream_fldfilename_irrig hgrid="0.9x1.25">lnd/clm2/prescribed_data/LFMIP-pdLC-SST.H2OSOI.0.9x1.25.20levsoi.natveg.1980-2014.MONS_climo.c190716.nc</stream_fldfilename_irrig>
+
+<irrig_mapalgo>nn</irrig_mapalgo>
+<irrig_tintalgo>nearest</irrig_tintalgo>
```

Comment:
- [ ] `irrig_tintalgo` must always be `nearest` for categorical variables like `irrig_method_patch`, for which interpolation isn't meaningful. Options to fix:
- Remove `irrig_tintalgo` as something that can be changed
- Silently use `nearest` to affect categorical variables
- Fail if something other than `nearest` is used but categorical variables are read from the streams file

Reply:
From conversation with @swensosc: For now, I will leave `irrig_tintalgo ` and `irrig_mapalgo` as namelist parameters, but only allow `nearest`/`nn`. This will provide the framework for eventually adding other options if users request them.